mcs/swagger.yml

```diff
@@ -536,6 +536,44 @@ paths:
       security: []
       tags:
         - UserAPI
+  /api/v1/profiling/start:
+    post:
+      summary: Start recording profile data
+      operationId: ProfilingStart
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/profilingStartRequest'
+      responses:
+        201:
```

Comment:
sould this be a `201` created or `200`? which one creates the file though?

Reply:
I used 201 because this is a `post` request, this operation creates a background process in Minio, we returned the status of that background process, then stopProfiling put everything on a file and send it back to the client